qiling/profiles/windows.ql

```diff
@@ -1,13 +1,17 @@
 [OS64]
-heap_address = 0x500000000
+heap_address = 0xfffff76000000000
```

Comment:
Is there any specific reason for this change?

Reply:
No specific reason relevant to Qiling. Was for my own testing purposes.